bitnami/rabbitmq/values.yaml

```diff
@@ -934,7 +938,7 @@ persistence:
   ## @param persistence.mountPath The path the volume will be mounted at
   ## Note: useful when using custom RabbitMQ images
   ##
-  mountPath: /bitnami/rabbitmq/mnesia
+  mountPath: /opt/bitnami/rabbitmq/.rabbitmq/mnesia
```

Comment:
Explanation: We were previously mounting over /bitnami/rabbitmq/mnesia, which is a symlink to /var/lib/rabbitmq/mnesia. At the same time, /var/lib/rabbitmq/mnesia was a symlink to /opt/bitnami/rabbitmq/.rabbitmq/mnesia. When mounting /opt/bitnami/rabbitmq/.rabbitmq (where we store the erlang cookie) as an emptydir, the folder /opt/bitnami/rabbitmq/.rabbitmq/mnesia disappears and therefore the mount fails.

This is a bit cumbersome but it's the only way for the bitnami/rabbitmq container to maintain compatibliity with rabbitmq-cluster-operator